modules/ui/fields/radio.js

```diff
@@ -223,13 +222,6 @@ export function uiFieldRadio(field, context) {
     }
 
 
-    function changeLayer(t, onInput) {
-        if (t.layer === '0') {
-            t.layer = undefined;
```

Comment:
The wiki [says](https://osm.wiki/Key:layer):

> 0 should not be used explicitly

does this sentance need to be updated? `layer=0` [is used 40,000 times](https://taginfo.openstreetmap.org/tags/layer=0), but usage continues to decrease. This PR might change that trend. 

just double checking so that we don't end up with [another situation like this](https://osm.wiki/ID/Controversial_Decisions)